index.html

```diff
@@ -1,9 +1,9 @@
 <!doctype html>
-
+<container>
 <html lang="en">
```

Comment:
you will never wrap a container around the html tags or the head tags, everything for styling and your HTML markup will always go inside the body tags